commercial/app/commercial/CommercialLifecycle.scala

```diff
@@ -22,12 +23,43 @@ trait CommercialLifecycle extends GlobalSettings with Logging with ExecutionCont
     MoneyBestBuysRefresh
   )
 
-  private def recordEvent(feedName: String, eventName: String, maybeDuration: Option[Duration]): Unit = {
-    val key = s"${feedName.toLowerCase.replaceAll("[\\s/]+", "-")}-$eventName-time"
-    val duration = maybeDuration map (_.toMillis.toDouble) getOrElse -1d
-    CommercialMetrics.metrics.put(Map(s"$key" -> duration))
+
+  private lazy val metricMap = Map(
+    "fetch-failure" -> AkkaAgent(0.0),
+    "fetch-success" -> AkkaAgent(0.0),
+    "parse-failure" -> AkkaAgent(0.0),
+    "parse-success" -> AkkaAgent(0.0)
+  )
```

Comment:
Going with the individual agents for ease of implementation and shouldnt be too much of an issue with only 4 agents